tools/protoxform_test_helper.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+from run_command import runCommand
+
+import logging
+import os
+import re
+import sys
+
+
+def PathAndFilename(label):
+  if label.startswith('/'):
+    label = label.replace('//', '/', 1)
+  elif label.startswith('@'):
+    label = re.sub(r'@.*/', '/', label)
+  else:
+    return label
+  label = label.replace(":", "/")
+  splitted_label = label.split('/')
+  return ['/'.join(splitted_label[:len(splitted_label) - 1]), splitted_label[-1]]
+
+
+def GoldenProtoFile(path, file, version):
+  base = "./"
+  base += path + "/" + file + "." + version + ".gold"
+  return os.path.abspath(base)
+
+
+def ResultProtoFile(path, file, version):
+  base = "./bazel-bin"
+  base += path + "/protos" + path + "/" + filename + "." + version + ".proto"
+  return os.path.abspath(base)
+
+
+def Diff(result_file, golden_file):
+  command = 'diff -u '
+  command += result_file + ' '
+  command += golden_file + ' '
+  status, stdout = runCommand(command)
+  return [status, stdout]
+
+
+def RunV3Alpha(path, filename):
+  message = ""
+  golden_path_v3 = GoldenProtoFile(path, filename, 'v3alpha')
+  test_path_v3 = ResultProtoFile(path, filename, 'v3alpha')
+
+  status, msg = Diff(test_path_v3, golden_path_v3)
+
+  if status != 0:
+    message = '\n'.join([str(line) for line in msg])
+
+  return message
+
+
+def RunV2(path, filename):
```

Comment:
`RunV2` and `Runv3Alpha` are very similar, can they just be a single parameterized function?